cmd/wallet-web/src/store/modules/options.js

```diff
@@ -138,50 +139,52 @@ export default {
         readCredentialManifests = readManifests();
       }
 
-      const optValue = (opt) => (opt in agentOpts ? agentOpts[opt] : defaultAgentStartupOpts[opt]);
+      const chooseOptValue = (opt) =>
```

Comment:
Any specific reason to rename this ?

Reply:
Since it is a function, it sound cleaner to me to use the verb in its name. Essentially, this function chooses which option to use @rolsonquadras @sudeshrshetty

Reply:
fine by me :+1: